### ISSUE TITLE
Users should be allowed to create HTTP verbs when creating APIs

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6854]
Users should be allowed to create HTTP verbs when creating APIs, currently, it's a two-step process

Issues raised by Usability Testing candidates

**Comment by chanaka3d** [comment]
With the 3.x UX design, the API creation steps are simplified. As a result, Resource creation has been moved to a different step that can be performed separately.
